src/types.ts

```diff
@@ -80,3 +80,12 @@ export type Mode =
   | "snap-polygon"
   | "split-route"
   | "street-view";
+
+export function schemaTitle(schema: Schema): string {
+  return {
+    v1: "Schema design",
```

Comment:
```suggestion
    v1: "Basic design mode",
```

Reply:
Isn't the confusion that we should be calling it Scheme design? they're designing a scheme not a schema.

I'd say 
``` 
v1: Scheme Design
v2: Experimental Scheme Design
```
might be what we're looking for. 

Otherwise this change looks good

Reply:
Ahh, what an awful typo -- I meant scheme, but we have two words in active use too similar >_<   Thank you, fixed. Also capitalized as suggested.

![Screenshot from 2023-07-07 09-54-47](https://github.com/acteng/atip/assets/1664407/6da1b6af-1031-43e2-8320-f2b026e73d3d)